src/components/Ariadne/CircularViewer/CircularViewer.tsx

```diff
@@ -28,6 +34,8 @@ export const CircularViewer = ({ sequence, size, annotations }: Props) => {
         className={`stroke-current`}
         width={sizeX}
         height={sizeY}
+        transform={`rotate(${scrollVal})`}
+        onWheel={(e) => handleScroll(e)}
```

Comment:
Any reason we went with onWheel instead of something like onScroll?

Reply:
From stackoverflow: "onwheel : Occurs when the mouse wheel is rolled up or down over an element. onscroll Occurs when an element's scrollbar is being scrolled"